src/spec/spec.cr

```diff
@@ -209,7 +209,7 @@ OptionParser.parse! do |opts|
     exit
   end
   opts.on("-v", "--verbose", "verbose output") do
-    Spec.formatter = Spec::VerboseFormatter.new
+    Spec.formatters << Spec::VerboseFormatter.new
```

Comment:
Oh, didn't see this. You can change it to push the VerboseFormatter if the `-v` option is passed, otherwise push the DotFormatter. However, I don't see how a user running specs would choose to add other formatters. I guess in their code that would have to do it, in a spec_helper.cr, right?